apps/crc_interactive.py

```diff
@@ -85,12 +92,7 @@ def _validate_arguments(self, args: Namespace) -> None:
         """
 
         # Check wall time is between limits, enable both %H:%M format and integer hours
-        if args.time.isdecimal():
-            check_time = int(args.time)
-        else:
-            check_time = (
-                datetime.strptime(args.time, '%H:%M').hour +
-                datetime.strptime(args.time, '%H:%M').minute / 60)
+        check_time = int(args.time)
```

Comment:
This type cast does not work. Python cannot automatically cast from time objects to integers. You need to handle hours, minutes, and seconds manually.

```
check_time = args.time.hour + args.time.minute / 60 + args.time.second / 3600
```